src/nxt_fs.c

```diff
@@ -11,9 +11,10 @@ static nxt_int_t nxt_fs_mkdir(const u_char *dir, mode_t mode);
 nxt_int_t
 nxt_fs_mkdir_all(const u_char *dir, mode_t mode)
 {
-    char    *start, *end, *dst;
-    size_t  dirlen;
-    char    path[PATH_MAX];
+    char       *start, *end, *dst;
+    size_t     dirlen;
+    nxt_int_t  ret;
+    char       path[PATH_MAX];
```

Comment:
Personally I'm not fussed about having `Christmas tree` variable type ordering and it was 'wrong' before, so meh...

( I could also quite happily live without the alignment, and the multiple variable declaration on a single line for that matter, it would make changes like this simpler)